spec/intl.html

```diff
@@ -40,23 +40,77 @@ <h1><a href="https://tc39.es/ecma402/#sec-case-sensitivity-and-case-mapping">Cas
+          An ECMAScript implementation that includes the ECMA-402 Internationalization API <b>must</b> be time zone aware.
+          However, it is recommended that all ECMAScript implementations be time zone aware, including those that do not implement the ECMA-402 Internationalization API.
+          Furthermore, even non-time zone aware implementations that do not support the entire IANA Time Zone Database are still recommended to use IANA Time Zone Database names as identifiers to represent time zones.
+        </p>
+
+        <p>
+          The IANA Time Zone Database is typically updated between five and ten times per year, so it is recommended to use the best available current and historical time zone information.
+          This information includes which identifiers are supported, the primary time zone identifier associated with any identifier, and the UTC offsets and transitions associated with any Zone.
+        </p>
+
+        <p>
+          New Zone identifiers can be added to the IANA Time Zone Database, for example when one part of a country starts observing Daylight Saving Time differently from other parts, or when a new country declares independence.
+          ECMAScript implementations are recommended to include newly-added identifiers as soon as possible into the results of AvailableTimeZoneIdentifiers.
+          Such prompt action ensures that ECMAScript programs receiving those identifiers from an external source (including the host's operating system) will be able to recognize and calculate using the new time zone.
+        </p>
+
+        <p>
+          It is recommended that implementations maintain a fully consistent copy of the IANA Time Zone Database for the lifetime of each agent.
+          If implementations do revise time zone information during the lifetime of an agent, then it is recommended that changes to time zone data, including which identifiers are supported, the primary time zone identifier associated with any identifier, and the UTC offsets and transitions associated with any Zone, can be incorporated into an agent only if they are consistent with results already observed by all ECMAScript code that can reach that agent.
+          For example, it is recommended that a new identifier can be incorporated only if no ECMAScript code has already tried to use it, and it is recommended that replacement of a primary identifier with a Link to a different identifier can only be incorporated only if no ECMAScript code has already resulted in resolving it as primary.
+          The complexity of maintaining such invariants is why it is recommended that implementations maintain a fully consistent copy of the IANA Time Zone Database for the lifetime of each agent.
```

Comment:
Small nitpick: if these are recommendations, then they're not invariants. Should we make them into invariants, i.e. "if implementations do revise time zone information during the lifetime of an agent, then it is ~~recommended~~ _required_ that..."?

Reply:
That's fine, but then we shouldn't call them invariants in this line either.

Reply:
Agree. New text I just pushed: 
> Because supporting the recommendations above can be difficult, it is recommended that implementations maintain a fully consistent copy of the IANA Time Zone Database for the lifetime of each agent.

Reply:
Actually, switched to "Due to the complexity of supporting these recommendations," here and in the other two places where similar language was used.